Build on OSX with clang 11 reports warnings

Two warnings during build on OSX with clang
```
g++ --version
Configured with: --prefix=/Applications/Xcode.app/Contents/Developer/usr --with-gxx-include-dir=/Library/Developer/CommandLineTools/SDKs/MacOSX.sdk/usr/include/c++/4.2.1
Apple clang version 11.0.0 (clang-1100.0.33.12)
Target: x86_64-apple-darwin18.7.0
Thread model: posix
InstalledDir: /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
```

1st:
```
INFO: From Compiling neuropods/multiprocess/multiprocess.cc:
neuropods/multiprocess/multiprocess.cc:37:9: warning: ISO C++11 does not allow conversion from string literal to 'char *' [-Wwritable-strings]
        "neuropod_multiprocess_worker",
```
here explicit cast helps

2nd:
```
INFO: From Compiling neuropods/multiprocess/ipc_control_channel.cc:
In file included from neuropods/multiprocess/ipc_control_channel.cc:10:
./neuropods/multiprocess/shm_tensor.hh:39:7: warning: unused function 'get_next_aligned_offset' [-Wunused-function]
void *get_next_aligned_offset(void *base)
```

Compiler warning is not correct, function is used. Function is defined in header and making it "inline" fixes warning and is reasonable at the same time.

I can provide pull request

--- a/source/neuropods/multiprocess/multiprocess.cc
+++ b/source/neuropods/multiprocess/multiprocess.cc
@@ -34,7 +34,7 @@ pid_t start_worker_process(const std::string &control_queue_name)
 {
     pid_t child_pid;
     char * argv[] = {
-        "neuropod_multiprocess_worker",
+        const_cast<char*>("neuropod_multiprocess_worker"),
         const_cast<char *>(control_queue_name.c_str()),
         NULL
     };

--- a/source/neuropods/multiprocess/shm_tensor.hh
+++ b/source/neuropods/multiprocess/shm_tensor.hh
@@ -36,7 +36,7 @@ struct __attribute__((__packed__)) shm_tensor
 };
 
 // TODO(vip): Use std::align
-void *get_next_aligned_offset(void *base)
+inline void *get_next_aligned_offset(void *base)
 {
     // We want to find an offset such that the data will be 64 byte aligned
     uint64_t base_address   = reinterpret_cast<uint64_t>(base);

